src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,213 @@
+                        history.add(task);
+                        int length = history.size();
+
+                        System.out.println("Added task:");
+                        System.out.println(task);
+                        System.out.println("You have " + String.valueOf(length) + " tasks in the list");
+
+                        try {
+                            appendToFile(LOCAL_FILE, a + System.lineSeparator());
+                        } catch (IOException e) {
+                            System.out.println("Something went wrong: " + e.getMessage());
+                        }
+
+                        a = sc.nextLine();
+                        b = a.split(" ", 2);
+
+                    } else if (b[0].equals("event")) {
+                        String[] c = b[1].split(" /at ", 2);
+                        Event task = new Event(c[0], c[1]);
+                        history.add(task);
+                        int length = history.size();
+
+                        System.out.println("Added task:");
+                        System.out.println(task);
+                        System.out.println("You have " + String.valueOf(length) + " tasks in the list");
+
+                        try {
+                            appendToFile(LOCAL_FILE, a + System.lineSeparator());
+                        } catch (IOException e) {
+                            System.out.println("Something went wrong: " + e.getMessage());
+                        }
+
+                        a = sc.nextLine();
+                        b = a.split(" ", 2);
+                    } else if (b[0].equals("delete")) {
+                        int taskIndex = Integer.valueOf(b[1]);
+                        Task removed = history.get(taskIndex - 1);
+
+                        history.remove(taskIndex - 1);
+                        int length = history.size();
+
+                        System.out.println("Ok! I have removed this task:");
+                        System.out.println(removed);
+                        System.out.println("Now you have " + String.valueOf(length) + " tasks in the list.");
+
+                        try {
+                            appendToFile(LOCAL_FILE, a + System.lineSeparator());
+                        } catch (IOException e) {
+                            System.out.println("Something went wrong: " + e.getMessage());
+                        }
+
+                        a = sc.nextLine();
+                        b = a.split(" ", 2);
+                    } else {
+                        throw new DukeException("I do not know what you want to do!");
+                    }
+                } catch (DukeException error) {
+                    System.out.println(error);
+
+                    a = sc.nextLine();
+                    b = a.split(" ", 2);
+                }
+            }
+            System.out.println("Bye! Hope to see you again soon!");
+            sc.close();
+
+    }
+}
```

Comment:
Perhaps you could apply more OOP by extracting out closely related code as classes, rather than letting the Main method handle everything.